modules/nf-core/pydamage/analyze/main.nf

```diff
@@ -4,34 +4,52 @@ process PYDAMAGE_ANALYZE {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/pydamage:0.70--pyhdfd78af_0' :
-        'biocontainers/pydamage:0.70--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/pydamage:1.0--pyhdfd78af_0' :
+        'biocontainers/pydamage:1.0--pyhdfd78af_0' }"
 
     input:
     tuple val(meta), path(bam), path(bai)
 
     output:
-    tuple val(meta), path("pydamage_results/*_pydamage_results.csv"), emit: csv
+    tuple val(meta), path("${prefix}_pydamage_results.csv"), emit: csv
     path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    prefix = task.ext.prefix ?: "${meta.id}"
     """
+    export NUMBA_CACHE_DIR=./tmp
+    export MPLCONFIGDIR=./tmp
+
     pydamage \\
         analyze \\
         $args \\
         -p $task.cpus \\
         $bam
 
-    mv pydamage_results/pydamage_results.csv pydamage_results/${prefix}_pydamage_results.csv
+    mv pydamage_results/pydamage_results.csv ${prefix}_pydamage_results.csv
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         pydamage: \$(pydamage --version | sed -n 's/pydamage, version \\(.*\\)/\\1/p')
     END_VERSIONS
     """
+
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    export NUMBA_CACHE_DIR=./tmp
+    export MPLCONFIGDIR=./tmp
+
```

Comment:
```suggestion
```

Reply:
This is required to make the versions command output not have a warning in it.
I'll add a comment.